packages/material-ui/src/CardMedia/CardMedia.js

```diff
@@ -47,11 +56,17 @@ const CardMedia = React.forwardRef(function CardMedia(props, ref) {
       style={composedStyle}
       src={isMediaComponent ? image || src : undefined}
       {...other}
-    />
+    >
+      {children}
+    </Component>
   );
 });
 
 CardMedia.propTypes = {
+  /**
+   *
```

Comment:
Oh sure. 

Children should always be explicit. There are a lot of misconceptions about this prop which mostly stem from people assuming they "just work". This is bascially true for every component in this library but only because our components are very close to the DOM. The bigger the component the less obvious what kind of children are handled if any.